exceptiongroup==1.1.1; python_version < '3.11'
iniconfig==2.0.0
mfrc522==0.0.7
mysql-connector-python==8.0.33
packaging==23.1
pluggy==1.0.0
protobuf==3.20.3
pytest==7.3.1
rpi.gpio==0.7.1
spidev==3.6
tomli==2.0.1; python_version < '3.11'
